csi-sidecars.mk

```diff
@@ -3,8 +3,10 @@ EXTERNAL_PROVISIONER_VERSION = 3.1.0
 EXTERNAL_RESIZER_VERSION = 1.4.0
 NODE_DRIVER_REGISTRAR_VERSION = 2.4.0
 LIVENESSPROBE_VERSION = 2.6.0
+EXTERNAL_SNAPSHOTTER_VERSION = 5.0.1
 CSI_SIDECARS = \
```

Comment:
here too, you can use `6.0.1`

Reply:
Majorly all the sidecars require an upgrade and I would like to take it up in a follow-up PR upgrading all the sidecars.